migration/migration.go

```diff
@@ -163,3 +175,93 @@ func recalculateBloomFilters(txn db.Transaction) error {
 		}
 	}
 }
+
+var trieNodeBuckets = map[db.Bucket]struct {
```

Comment:
`trieNodeBuckets` is updated and used as a persistent storage across multiple invocations of  `changeTrieNodeEncoding`. So it needs to be global.

trieNode is used in the package test. so unless we want to redefine it in the tests, we should leave it a package level definition.

Reply:
My bad I missed that. 

For a future PR, what do you think about a `Migration` interface with a single `Migrate(txn)` method? We could use the `http.HandlerFunc` pattern to avoid needing to implement the interface with a struct when unnecessary.

Reply:
> For a future PR, what do you think about a Migration interface with a single Migrate(txn) method?

Hmm, sounds really good. :+1:

Reply:
Issue created #884